src/scss/_about.scss

```diff
@@ -0,0 +1,49 @@
+//---------------Break Points---------------//
+//50em (800px)
+//61.25em (980px)
+
+.about {
+  background-color: $color-blue;
+  text-align: center;
+  padding: 4em 0; //64px 0
+}
+
+.about__heading {
```

Comment:
Just as an FYI, you can save some keystrokes and also make this a bit more readable (to me anyway) by nesting your components like so:

```
.about {
  &__heading {}
}
```
This will output CSS that looks like this:

```
.about {}
.about__heading {}
```

Another benefit is that if you ever decide to rename the `[block]` portion of your component name, you only have to change it in one place.